Library/Formula/kibana.rb

```diff
@@ -0,0 +1,71 @@
+class Kibana < Formula
+  homepage 'https://www.elastic.co/products/kibana'
+  url 'https://download.elastic.co/kibana/kibana/kibana-4.0.2-darwin-x64.tar.gz'
+  version '4.0.2'
+  sha1 'ce6136d826cf8ac1da71abea419f2f5220055933'
+  head 'https://github.com/elastic/kibana.git'
+
+  def install
+    # install base files
+    prefix.install %w[ node src ]
+
+    # setup etc path
+    etc_path.mkpath
+
+    # setup var path
+    var_path.mkpath
```

Comment:
I guess if it's necessary to actually symlink stuff back to the `prefix` it's useful, but even then you can safely afford to cut the `def`s in half in number. Just `prefix.install_symlink var_path/"plugins"`. Don't need a def there.